Kickstarter-iOS/Views/Cells/SettingsCurrencyCell.swift

```diff
@@ -0,0 +1,54 @@
+import KsApi
+import Library
+import Prelude
+
+final class SettingsCurrencyCell: UITableViewCell, NibLoading, ValueCell {
+  @IBOutlet fileprivate weak var titleLabel: UILabel!
+  @IBOutlet fileprivate weak var currentCurrencyLabel: UILabel!
+  @IBOutlet fileprivate weak var separatorView: UIView!
+
+  private let viewModel: SettingsCurrencyCellViewModelType = SettingsCurrencyCellViewModel()
+
+  func configureWith(value: SettingsCellValue) {
+    switch value.cellType {
+    case SettingsAccountCellType.currency:
+      NotificationCenter.default
```

Comment:
I think there might be a way to do this without notifications. What if the `dataSource` passed the value of the currency in as part of the `SettingsValueCellValue`?